packages/lab/src/input-next/InputNext.css

```diff
@@ -30,17 +25,18 @@
 
 .saltInputNext:hover {
   --inputNext-borderStyle: var(--salt-editable-borderStyle-hover);
+  --inputNext-borderColor: var(--salt-editable-borderColor-hover);
 
```

Comment:
we don't seem to be using these border color states. Am I missing something?
```
  --inputNext-borderColor: var(--salt-status-error-borderColor);
  --inputNext-borderColor-active: var(--salt-status-error-borderColor);
  --inputNext-borderColor-hover: var(--salt-status-error-borderColor);
  --inputNext-borderColor-focused: var(--salt-status-error-borderColor);
```

Reply:
thanks, fixed